joomla/joomla-cms
# Extend the message to explain what is happening

Pull Request for Issue #16238

### Summary of Changes

In some broken hosting envoirments both mime checker tools are not enabled. In that case show that message with more extended info.

### Testing Instructions

- Try this on a broken hosting.
- confirm the extended message

- review the text.

### Expected result

Extended text explaining what is happening

### Actual result

Just a general message which is not longer match the usage.

### Documentation Changes Required

None
